app/src/main/java/openfoodfacts/github/scrachx/openfood/views/SplashActivity.java

```diff
@@ -40,6 +41,13 @@ public void onCreate(Bundle icicle) {
         settings = getSharedPreferences("prefs", 0);
         mAdditiveDao = Utils.getAppDaoSession(this).getAdditiveDao();
 
+        ConnectivityManager connectivityManager=(ConnectivityManager)getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo netinfo=connectivityManager.getActiveNetworkInfo();
+        if (!(netinfo!=null && netinfo.isConnected()))
+        {
+            Toast.makeText(getApplicationContext(),"No Internet Access",Toast.LENGTH_SHORT).show();
+        }
+
```

Comment:
This code is repeated across the files of this commit. Can you please move out the redundant code to a helper function in other package (i.e. utils)?